tests/sqlparser_databricks.rs

```diff
@@ -366,3 +366,87 @@ fn data_type_timestamp_ntz() {
         s => panic!("Unexpected statement: {s:?}"),
     }
 }
+
+#[test]
+fn parse_table_time_travel() {
+    let version = "2018-10-18T22:15:12.013Z".to_string();
+    let sql = format!("SELECT 1 FROM t1 TIMESTAMP AS OF '{version}'");
+    let select = databricks().verified_only_select(&sql);
```

Comment:
for the tests we can use
```rust
let dialect = all_dialecs_where(|d| d.supports_timestamp_versioning());
```

Reply:
Addressed in https://github.com/apache/datafusion-sqlparser-rs/pull/2134/commits/270c41b9ac6c46b396dc47ec435f1a37afe6e883